scripts/ci/before_install.sh

```diff
@@ -53,10 +53,10 @@ fi
 
 if [[ -n "$TRAVIS_TAG" ]]; then
   if [[ $(echo "$TRAVIS_TAG" | grep -E '^[0-9]+\.[0-9]+\.[0-9]+$') ]]; then
-    echo "export SNAP_CHANNEL=stable" >>env.sh
+    echo "export SNAP_CHANNEL=stable,v0/stable" >>env.sh
   else
-    echo "export SNAP_CHANNEL=beta" >>env.sh
+    echo "export SNAP_CHANNEL=beta,v0/beta" >>env.sh
```

Comment:
I guess we could also parse the major version and push to `vN` channel?

Reply:
that is a good point. properly future-proof. Will change.

Reply:
done